source/intro.Rmd

```diff
@@ -70,149 +63,206 @@ the methods described in this book:
+*number* — of which more below.  Or it can refer to the field of inquiry that
+studies these numbers.
+
+A *statistic* is a *number* that we can calculate from a larger collection of
+numbers we are interested in.  For example, table \@ref(tab:yearly-srp) has
+some yearly measures of "soluble reactive phosphorus" (SRP) from Lough Erne —
+a lake in Ireland [@zhou2000lake].
+
+```{python, echo=FALSE}
+import os.path as op
+import numpy as np
+import pandas as pd
+lake = pd.read_csv(op.join('data', 'lough_erne.csv'))
+yearly_srp = lake.loc[:, ['Year', 'SRP']].copy()
+```
+
+```{r, yearly-srp, echo=FALSE}
+ketable(py$yearly_srp, caption = "Soluble Reactive Phosphorus in Lough Erne")
+```
+
+We may want to *summarize* this set of SRP measurements. For example, we could
+add up all the SRP values to give the *total*.  We could also divide the total
+by the number of measurements, to give the *average*.  Or we could measure the
+spread of the values by finding the *minimum* and the *maximum* — see table
+\@ref(tab:srp-stats). All these numbers are *descriptive statistics*, because
+they are summaries that *describe* the collection of SRP measurements.
+
+
+```{python, echo=FALSE}
+srp = np.array(yearly_srp['SRP'])
+srp_stats = pd.DataFrame({'Descriptive statistics for SRP': pd.Series({
+    'Total': np.sum(srp),
+    'Mean': np.mean(srp),
+    'Minimum': np.min(srp),
+    'Maximum': np.max(srp)})})
+```
+
+```{r, srp-stats, echo=FALSE}
+ketable(head(py$srp_stats), caption = "Statistics for SRP levels")
+```
+
+Descriptive statistics are nothing new to you; you have been using many of them
+all your life.
+
+We can calculate other numbers that can be useful for drawing *conclusions* or
+*inferences* from a collection of numbers; these are *inferential statistics*.
+Inferential statistics are often probability values that give the answer to
+questions like "What are the chances that ...".
+
+```{r, echo=FALSE}
+ysrp <- py$yearly_srp
+pre_1990 <- round(mean(ysrp[ysrp$Year < 1990, 'SRP']), 1)
+post_1990 <- round(mean(ysrp[ysrp$Year > 1990, 'SRP']), 1)
+```
+
+For example, imagine we suspect there was some environmental change in 1990. We
+see that the average SRP value before 1990 was `r pre_1990` and the average SRP
+value after 1990 was `r post_1990`.  That gives us a difference in the average
+of `r post_1990 - pre_1990`. But, could this difference be due to chance
+fluctuations from year to year?  Were we just unlucky in getting a few larger
+measurements in later years?  We could use methods that you will see in this
+book to calculate a *probability value* for that question.  The probability
+value is an *inferential* statistic, because we can use it to draw an
+*inference* about the measures.
+
+Inferential statistics use descriptive statistics as their input. Inferential
+statistics can be used for two purposes: to aid scientific *understanding* by
+estimating the probability that a statement is true or not, and to aid in
```

Comment:
A probability is usually only attached to one of these things (true / not true); and we have to be careful here not to let people think that we can guess whether things are true.

Reply:
Well - but we are estimating the probability - no?